.bazelci/presubmit.yml

```diff
@@ -0,0 +1,42 @@
+---
+platforms:
+  ubuntu1804:
+    build_flags:
+    - "--build_tag_filters=-nolinux"
+    build_targets:
+    - "..."
+    test_flags:
+    - "--features=race"
```

Comment:
Let's also remove race testing. This makes the test a lot heavier. Gazelle has almost no concurrency.

Reply:
Done